tests/setup_files/supervisord/conf.d/tftpd.conf

```diff
@@ -0,0 +1,3 @@
+[program:tftpd]
+command=/usr/sbin/in.tftpd -L -u root -s /srv/tftpboot
```

Comment:
are you sure that -u root is necessary here?

Reply:
To my research yes. Otherwise it did not start up. Ideas?

Reply:
You're correct, if your supervisor isn't running as root, it's necessary, because otherwise in.tftpd wouldn't be allowed to open the privileged tftp port 69/udp.